action.yml

```diff
@@ -0,0 +1,212 @@
+name: Devin AI Action
+description: |
+  Creates a new Devin.ai session with a given prompt or playbook. Designed to be used in slash commands and can post status updates back to PR comments.
+branding:
+  icon: cpu
+  color: purple
+
+inputs:
+  comment-id:
+    description: "Comment ID (optional, for reply chaining and context)"
+    required: false
+  issue-number:
+    description: "Issue number (optional, for context)"
+    required: false
+  runbook-id:
+    description: "Runbook/Playbook ID (optional, for structured workflows)"
```

Comment:
Let's rename everywhere to "playbook macro" and let's specify here in the description that it should start with "!" - e.g `!my_playbook`